Fixes

ItsMeLeeson opened the pull request:
SongInfo.cs - Uses new web socket addresses for JPOP and KPOP
Fixes missing song title, album, and artist(s) from being displayed
Fixes Discord rich presence

WebStreamPlayer.cs - Change catch exception object to catch any type
Fixes new songs not playing after current one finishes

ItsMeLeeson committed:
Use new web socket addr, and fix new song playback

SongInfo.cs - Uses new web socket addresses for JPOP and KPOP
Fixes missing song title, album, and artist(s) from being displayed
Fixes Discord rich presence

WebStreamPlayer.cs - Change catch exception object to catch any type
Fixes new songs not playing after current one finishes

robflop commented:
Hi, just wanna let you know that this repo hasn't been maintained in a long time and I doubt it will be in the future, so the chances of a PR actually being reviewed/merged are low.

The player that does get maintained is located at https://github.com/LISTEN-moe/desktop-app, although that one is based on Electron. If you'd like, you can have a look there.

If you don't like Electron, you could maybe try to maintain a private fork of this C# player and update it to keep working properly.

ItsMeLeeson committed:
Implement Keep Form On Top option to context menu

ItsMeLeeson commented:
#39 added

iCrawl commented:
I would actually merge and publish this, BUT, in your last commit you mixed spaces and tabs, if you could correct that that'd be nice.

ItsMeLeeson committed:
Fix indenting from previous commit

Oops I'll blame Visual Studio

ItsMeLeeson commented:
Sorry about that fixed it now

ItsMeLeeson committed:
Update MUSIC_LINK, and add null album covers

Use new music link url so clicking an album cover doesn't lead to an invalid page.
For albums without a cover image the missing image will be used. This is so that users can still click the cover to be taken to the listen.moe album entry.
